predict.py

```diff
@@ -0,0 +1,302 @@
+import sys
```

Comment:
Please add the following copyright and license notice in the header:

Copyright (c) Facebook, Inc. and its affiliates.
All rights reserved.

This source code is licensed under the license found in the
LICENSE file in the root directory of this source tree.